<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGui::MainConfigWizardPage</class>
 <widget class="QWidget" name="QtGui::MainConfigWizardPage">
  <property name="windowTitle">
   <string>General</string>
  </property>
  <property name="windowIcon">
   <iconset theme="internet-web-browser" resource="../resources/syncthingwidgetsicons.qrc">
    <normaloff>:/icons/hicolor/scalable/apps/internet-web-browser.svg</normaloff>:/icons/hicolor/scalable/apps/internet-web-browser.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="invalidConfigLabel">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="cfgCurrentlyRunningRadioButton">
     <property name="text">
      <string>Configure Syncthing Tray for currently running Syncthing instance</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="cfgLauncherExternalRadioButton">
     <property name="text">
      <string>Start installed Syncthing application via Syncthing Tray</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="cfgLauncherBuiltInRadioButton">
     <property name="text">
      <string>Start Syncthing application that is built into Syncthing Tray</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="cfgSystemdUserUnitRadioButton">
     <property name="text">
      <string>Start Syncthing via systemd user-unit (enables and starts the unit %1)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="cfgSystemdSystemUnitRadioButton">
     <property name="text">
      <string>Start Syncthing via systemd system-unit (enables and starts the unit %1)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="cfgNoneRadioButton">
     <property name="text">
      <string>Keep settings for how to connect to Syncthing and start Syncthing as they are</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="enableSystemdIntegrationCheckBox">
     <property name="text">
      <string>Enable systemd-integration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="warningLabel">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Click next to configure Syncthing Tray according to the selected options. You can also reconduct the checks the available options are based on by heading back. If you cancel the wizard, no configuration changes will be made.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/syncthingwidgetsicons.qrc"/>
 </resources>
 <connections/>
</ui>
